PhysicsTools/PyTorch/test/testPipeline.py

```diff
@@ -0,0 +1,114 @@
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string(args.backend)
+    ),
+)
+
+# JIT models
+process.JitClassificationProducer = process.JitClassificationProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    modelPath = cms.FileInPath(args.classificationModelPath),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string(args.backend)
+    ),
+)
+process.JitClassificationProducerCpu = process.JitClassificationProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    modelPath = cms.FileInPath(args.classificationModelPath),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string("serial_sync")  # force serial backend to emulate heterogeneous execution
+    ),
+)
+process.JitRegressionProducer = process.JitRegressionProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    modelPath = cms.FileInPath(args.regressionModelPath),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string(args.backend)
+    ),
+)
+process.JitRegressionProducerCpu = process.JitRegressionProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    modelPath = cms.FileInPath(args.regressionModelPath),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string("serial_sync")  # force serial backend to emulate heterogeneous execution
+    ),
+)
+
+# AOT models
+regressionModelPath = args.regressionModelPathCpu
+if args.backend == "cuda_async":
+    regressionModelPath = args.regressionModelPathCuda
+process.AotRegressionProducer = process.AotRegressionProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    modelPath = cms.FileInPath(regressionModelPath),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string(args.backend)
+    ),
+)
+classificationModelPath = args.classificationModelPathCpu
+if args.backend == "cuda_async":
+    classificationModelPath = args.classificationModelPathCuda
+process.AotClassificationProducer = process.AotClassificationProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    modelPath = cms.FileInPath(classificationModelPath),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string(args.backend)
+    ),
+)
+
+process.CombinatoricsProducer = process.CombinatoricsProducerStruct.clone(
+    inputs = cms.InputTag('DataProducer'),
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string(args.backend)
+    ),
+)
+
+# schedule the modules
+process.path = cms.Path(
+    process.DataProducer +
+    process.JitClassificationProducer +
+    # process.JitClassificationProducerCpu +
+    # process.JitRegressionProducer +
+    process.JitRegressionProducerCpu +
+
+    # disable since no proper support is available yet
+    # process.AotRegressionProducer +
+    # process.AotClassificationProducer +
+
+    process.CombinatoricsProducer
+)
+
+process.schedule = cms.Schedule(
+    process.path
+)
```

Comment:
This is not necessary. In absence of `process.schedule` the framework will run all Paths.
```suggestion
```